src/errors/FetchError.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+class FetchError extends Error {
+  innerError: Error
+
+  getMessage = (endpointName: string, innerError: Error): string =>
+    `FetchError: Failed to load the request for the ${endpointName} endpoint. ${innerError.message}`
+
+  constructor (params: {endpointName: string, innerError: Error}) {
+    super()
+    this.message = this.getMessage(params.endpointName, params.innerError)
+    this.innerError = params.innerError
+
+    if (Error.captureStackTrace) {
```

Comment:
Did you check whether the line numbers are oke in the stack traces of this error? 

If not then this error should be structured exactly like this error: 

https://github.com/Integreat/integreat-react-native-app/blob/5ab42143683999425ef5fe2d428e262f6a65a10e/src/modules/error/ContentNotFoundError.js#L13-L12

(Move captureStackTrace directly below the super)

Reply:
The ContentNotFoundError is tested afaik